pytorch_blade/torch_blade/onnx_backends/backend_conversion.py

```diff
@@ -57,6 +57,16 @@ def _build_onnx_engine(subgraph, engine_build_func, q_info=None, dynamic_setting
     if dynamic_settings is not None:
         dynamic_shapes = dynamic_settings
         dynamic_axes = dynamic_shapes[0].dynamic_setting
+        min_shape = dynamic_shapes[0].min_shape
+        max_shape = dynamic_shapes[0].max_shape
+        opt_shapes = dynamic_shapes[0].opt_shapes
+        logger.info(
+            "Dynamic shape settings: \n"
+            "min_shape: {str(min_shape)},\n"
```

Comment:
missing the `f` prefix? btw, I usually don't log into multiple lines, feeling not good when `grep`ing.